apollo-router-core/src/error.rs

```diff
@@ -219,11 +219,20 @@ pub enum JsonExtError {
     InvalidFlatten,
 }
 
+#[derive(Error, Debug, Display, Clone)]
+pub enum ServiceBuildError {
+    /// Couldn't build Router Service: {0}
+    QueryPlannerError(QueryPlannerError),
+}
+
 /// Error types for QueryPlanner
 #[derive(Error, Debug, Display, Clone)]
 pub enum QueryPlannerError {
+    /// Couldn't instantiate QueryPlanner: {0}
```

Comment:
All error messages should begin with lower case.
Possible also needs to be changes in Display for BridgeErrors?

Reply:
fixed in 85ad45c7